cmd/kubectl-k8ssandra/register/command.go

```diff
@@ -69,14 +72,16 @@ func NewRegistrationExecutorFromRegisterClusterCmd(cmd cobra.Command) *Registrat
 		destName = registration.CleanupForKubernetes(srcContext)
 	}
 	return &RegistrationExecutor{
-		SourceKubeconfig: cmd.Flag("source-kubeconfig").Value.String(),
-		DestKubeconfig:   cmd.Flag("dest-kubeconfig").Value.String(),
-		SourceContext:    srcContext,
-		DestContext:      cmd.Flag("dest-context").Value.String(),
-		SourceNamespace:  cmd.Flag("source-namespace").Value.String(),
-		DestNamespace:    cmd.Flag("dest-namespace").Value.String(),
-		ServiceAccount:   cmd.Flag("serviceaccount-name").Value.String(),
-		Context:          cmd.Context(),
-		DestinationName:  destName,
+		SourceKubeconfig:   cmd.Flag("source-kubeconfig").Value.String(),
+		DestKubeconfig:     cmd.Flag("dest-kubeconfig").Value.String(),
+		SourceContext:      srcContext,
+		DestContext:        cmd.Flag("dest-context").Value.String(),
+		SourceNamespace:    cmd.Flag("source-namespace").Value.String(),
+		DestNamespace:      cmd.Flag("dest-namespace").Value.String(),
+		ServiceAccount:     cmd.Flag("serviceaccount-name").Value.String(),
+		OverrideSourceIP:   cmd.Flag("oride-src-ip").Value.String(),
+		OverrideSourcePort: cmd.Flag("oride-src-port").Value.String(),
```

Comment:
**Suggestion:** There's already a discrepancy between whether or not some names are shortened (`source-` vs `dest-`) and `oride-` would confuse me a lot as a short version of `override-`. Would it be possible to use the long version of this for clarity?

```suggestion
		OverrideSourceIP:   cmd.Flag("override-src-ip").Value.String(),
		OverrideSourcePort: cmd.Flag("override-src-port").Value.String(),
```